File: results/preliminary_analysis/baseline_cpp_results/cpp_source/question-46-pass-0.cpp
// Count of elements not divisible by any other elements of Array. 
 #include <bits/stdc++.h> 
 #define ll  long long int 
 using namespace std ;
 int count ( int a [ ] , int n ) {
 
 	int count = 0 ;
 	for ( int i = 0 ; i < n ; i++ ) {
 		for ( int j = i + 1 ; j < n ; j++ ) {
 			if ( a [ i ] % a [ j ] == 0 ) {
 				count++ ;
 				break ;
 			}
 		}
 	}
 	return count ;
 }
 int main ( ) {
 	int n ;
 	cin >> n ;
 	int a [ n ] ;
 	for ( int i = 0 ; i < n ; i++ ) {
 		cin >> a [ i ] ;
 	}
 	

File: results/preliminary_analysis/baseline_py_results/py_source/question-46-pass-0.py
# Count of elements not divisible by any other elements of Array. 
def count(a, n): 
    count = 0
    for i in range(n): 
        if a[i]!= 0: 
            count += 1
    return count 
  
# Function to find the number of elements that are 
# divisible by all other elements of Array. 
def find(a, n): 
    count = 0
    for i in range(n): 
        if a[i]!= 0: 
            count += 1
    return count 
